.github/workflows/news.yml

```diff
@@ -0,0 +1,99 @@
+name: Add News
+
+on:
+  issue_comment:
+    types:
+      - created
+  pull_request_target:
+    types:
+      - reopened
+      - opened
+      - synchronize
+
+jobs:
+  check:
+    if: >-
+      !github.event.repository.fork
+      && (
+        github.event.comment.body == '@conda-bot please add news'
+        || github.event_name == 'pull_request_target'
+      )
+
+    runs-on: ubuntu-latest
+    steps:
+      - name: Extract PR branch name
+        shell: bash
+        run: echo "##[set-output name=branch;]$(echo ${GITHUB_REF#refs/heads/})"
+        id: pr_branch_name
+      
+      - name: Extract PR number
+        shell: bash
+        run: echo ${{ github.event.pull_request.number }}
+        id: pr_number
+
+      - name: Add news
+        uses: conda/actions/check-cla@v23.1.0
+
+
+    build:
+      runs-on: ubuntu-latest
+      steps:
+
+      # - name: Create news file
+      #   shell: bash
+      #   run:
+      #     # Create a variable that concatenates the PR number and PR branch name
+      #     # Create a file using that variable as the filename
+      #     |
+      #     file_name="${{ steps.pr_number }}-${{ steps.pr_branch_name }}"
+      #     touch $file_name
+
+      # - name: Populate file
+      #   shell: bash
+      #   run: |
+      #     echo -e "### Enhancements" >> $file_name
+      #     echo -e >> $file_name
+      #     echo -e "  * <news item>">> $file_name
+      #     echo -e >> $file_name
+      #     echo -e "### Bug fixes" >> $file_name
+      #     echo -e >> $file_name
+      #     echo -e "  * <news item>">> $file_name
+      #     echo -e >> $file_name
+      #     echo -e "### Deprecations" >> $file_name
+      #     echo -e >> $file_name
+      #     echo -e "  * <news item>">> $file_name
+      #     echo -e >> $file_name
+      #     echo -e "### Docs" >> $file_name
+      #     echo -e >> $file_name
+      #     echo -e "  * <news item>">> $file_name
+      #     echo -e >> $file_name
+      #     echo -e "### Other" >> $file_name
+      #     echo -e >> $file_name
+      #     echo -e "  * <news item>">> $file_name
```

Comment:
Kept this commented-out bit in here just for initial reference; this will all be deleted in a future commit.